server/utils/utils.ts

```diff
@@ -100,6 +100,11 @@ const jsonDtTo12HourTime = (dt: string): string => {
   return momentTime.isValid() ? momentTime.format('hh:mm a') : null
 }
 
+const removeDuplicates = (list: string[]): string[] => {
+  const deDuplicatedList = [...new Set(list.map(s => s))].map(s => s)
```

Comment:
What does `.map()` do here? Isn't this enough?

```
return [...new Set(list)]
```

Reply:
I think you're right. I've got some tests around this so can try a few alternatives in the morning.